pkgs/desktops/gnome/apps/gedit/default.nix

```diff
@@ -25,11 +25,11 @@
 
 stdenv.mkDerivation rec {
   pname = "gedit";
-  version = "43.alpha";
+  version = "43.1";
```

Comment:
Ugh, the NEWS says:

> * Come back to the source code of gedit 40.0 and continue again from there.
  It is based on the Tepl and Amtk libraries.

This is weird enough to me but not sure if we want to revert #160305 (see https://github.com/NixOS/nixpkgs/pull/160290#pullrequestreview-884117459)

Reply:
Dropped this update too